pkg/kube/longhorn-utils.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+#
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+LONGHORN_VERSION=v1.6.2
+
+longhorn_install() {
+    node_name=$1
+    logmsg "Installing longhorn version ${LONGHORN_VERSION}"
+    apply_longhorn_disk_config "$node_name"
+    lhCfgPath=/etc/lh-cfg-${LONGHORN_VERSION}.yaml
+    if ! grep -q 'create-default-disk-labeled-nodes: true' "$lhCfgPath"; then
+            sed -i '/  default-setting.yaml: |-/a\    create-default-disk-labeled-nodes: true' "$lhCfgPath"
```

Comment:
I see this code exists even in the previous version, so it is not expected to be changed in the scope of this PR. Nevertheless, I'm curious: Why do we make this change in runtime? Can't we just have an already fixed version in our repo? Or change it during the Docker build? Otherwise, it will not be that obvious for anyone checking the behaviour why the YAML file used in runtime differs from what we have in the source code.

Reply:
This remained separate from the longhorn provided yaml so that the base default yaml can be updated easily to newer versions.  Originally when this PR was submitted the yaml was still downloaded directly from the longhorn repo and patched before applying.  The yaml was only moved into pkg/kube to address earlier reviews and I overlooked manipulating the yaml during build time instead.